BBKoffieTuin/Assets/Scripts/GpsDebugUserInterface.cs

```diff
@@ -0,0 +1,25 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+public class GpsDebugUserInterface : MonoBehaviour
+{
+    [SerializeField] private GpsService gpsService;
+
+    public TMP_Text latitudeText;
+    public TMP_Text longitudeText;
+    public TMP_Text altitudeText;
+    public TMP_Text horizontalAccuracyText;
+    public TMP_Text timestampText;
```

Comment:
Waarom zijn dit geen SerializeFields?

Reply:
I'll change it but you don't have to be that harsh on 'debug scripts' they should be removed after this is finished and are of no importance